src/NHibernate/Linq/Visitors/SelectClauseNominator.cs

```diff
@@ -121,11 +121,11 @@ private bool IsRegisteredFunction(Expression expression)
 						   methodCallExpression.Object.NodeType != ExpressionType.Constant; // does not belong to parameter 
 				}
 			}
-			else if (expression.NodeType == (ExpressionType)NhExpressionType.Sum ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Count ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Average ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Max ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Min)
+			else if (expression is NhSumExpression ||
+			         expression is NhCountExpression ||
+			         expression is NhAverageExpression ||
+			         expression is NhMaxExpression ||
+			         expression is NhMinExpression)
```

Comment:
The default VS tooling does not support this well. As soon as we have written some code (or specifically, pasted some java code) and want to auto-fix the formatting, it switches those spaces alignements to tabs in the whole file.

Maybe I have mis-configured the tooling, but then I am unable to find the right option for preserving such formatting. (And I have tried with R# too.)